src/include/daos_srv/bio.h

```diff
@@ -86,6 +86,26 @@ struct bio_blob_hdr {
 	uuid_t		bbh_pool;
 };
 
+/*
+ * Current device state (health statistics). Periodically updated in
+ * bio_bs_monitor(). Used to determine faulty device status.
+ */
+struct bio_device_health_state {
+	uint64_t	 bhs_timestamp;
+	int		 bhs_bio_err;
```

Comment:
make it uint32_t and shift it to avoid hole in structure.  (for example, move it before 'bhs_temperature' and consume 64 bits together with 'temperature' and 'bits')

Reply:
ok I will update this.